python/test.py

```diff
@@ -3,7 +3,9 @@
 
 import numpy as np
 
-from libsonata import EdgeStorage, NodeStorage, Selection, SonataError
+from libsonata import EdgeStorage, NodeStorage, Selection, SonataError, SpikeReader
+
+from libsonata import SpikeReaderPopulation
```

Comment:
What about `SpikePopulation`? A little easier to read. Also, can we either get all imports on one line, or every import on a slngle one?

Reply:
In fact, `SpikeReaderPopulation` is an internal python type. Nobody should care about it, it is here only for test.
I don't know what you think about it. People cannot create it (no public ctor).

Reply:
Still, I think the `Reader` part is a little superfluous here. `SpikeReader` returning a `SpikePopulation` makes it a little easier than having to think about what a `SpikeReaderPopulation` is and how it differs from just a population of spikes?

Reply:
Suppose there is a writing API, would it be the same object as `SpikePopulation`, or would it be different?

Reply:
Could be the same.